tests/rptest/redpanda_cloud_tests/high_throughput_test.py

```diff
@@ -604,7 +604,7 @@ def swarm_start(node):
             if (len(swarm) - calc_alive_swarm_nodes(swarm)) > 1:
                 break
             elif _elapsed > FINISH_TIMEOUT_SEC and \
-                _connections < (self._advertised_max_client_count * 0.01):
```

Comment:
_connections is a list (this used to work, but _connections type was changed at some point), `_total` is the sum of all values in connections and is the correct lhs for this test